test-case/check-sof-logger.sh

```diff
@@ -47,23 +47,43 @@ if [[ ! -f $ldcFile ]]; then
 fi
 dlogi "Found file: $(md5sum $ldcFile|awk '{print $2, $1;}')"
 
-tmp_file=/tmp/$RANDOM.sof-logger.log
+rdnum=$RANDOM
+tmp_file=/tmp/$rdnum.logger.log
+err_tmp_file=/tmp/$rdnum.err_logger.log
 
 func_lib_check_sudo
 
 dlogi "Try to dump the dma trace log via sof-logger ..."
-dlogc "sudo $loggerBin -t -l $ldcFile -o $tmp_file 2>&1 &"
-sudo $loggerBin -t -l $ldcFile -o $tmp_file 2>&1 &
+# sof-logger errors will output to $err_tmp_file
+dlogc "sudo $loggerBin -t -l $ldcFile -o $tmp_file 2> $err_tmp_file &"
+sudo $loggerBin -t -l $ldcFile -o $tmp_file 2> $err_tmp_file &
 sleep 2
 dlogc "sudo pkill -9 $(basename $loggerBin)"
 sudo pkill -9 $(basename $loggerBin) 2> /dev/null
 
+# check if we get any sof-logger errors
+logger_err=`grep -i "error" $err_tmp_file`
+if [[ $logger_err ]]; then
+    dloge "No available log to export due to sof-logger errors."
+    exit 1
```

Comment:
please add `$err_tmp_file` dump

Reply:
Previously,  logger error info is duplicated in stdout and stderr, that why I don't dump here, but now, only the stderr.  This suggestion is good, and done and repushed